specification/metadata/metadata_schema.md

```diff
@@ -0,0 +1,64 @@
+# FOCUS Metadata Schema 
+Each FOCUS dataset must have a metadata about the schema associated with it. The schema metadata provides information about the structure of the data provided. In the event that the provider modifies the columns provided in the FOCUS dataset, metadata about the schema must be provided, including column definitions.  
```

Comment:
Does the schema belong to the provider or to the data generator?

I would go the simple path where we don't split the metadata between provider and data generator: there would be only data generator metadata, and then it would be the data generator schema.

I would tend to think